### PR TITLE
Fix up coverage

### PR DESCRIPTION
**Pull request by inverse**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,12 +5,12 @@ jobs:
...
-    - name: Run npm install, npm test
+        cmd: install
+    - name: Run yarn run test-with-coverage
       run: |
-        npm install
-        npm test
+        yarn run test-with-coverage
       env:
         CI: true
+        CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
```
Comment (inverse) [inline .github/workflows/test.yml]: @alexesprit any idea why this is not working? states it cannot find the env var :(

Comment (inverse) [reply .github/workflows/test.yml]: It's to push code coverage results to codacy so we can track how many lines are covered with tests etc so we can get a better feel of the tests

Comment (alexesprit) [reply .github/workflows/test.yml]: Does your token work locally? Maybe it's not valid or something else?

**Comment by inverse** [comment]
It works locally when I was using export

File: .github/workflows/test.yml (hunk 1 above)
Comment (inverse) [reply .github/workflows/test.yml]: It works locally for me :/

**Pull request closed without merging**